tests/components/totalconnect/common.py

```diff
@@ -0,0 +1,136 @@
+"""Common methods used across tests for TotalConnect."""
+from total_connect_client import TotalConnectClient
+
+from homeassistant.components.totalconnect import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.setup import async_setup_component
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+
+LOCATION_INFO_BASIC_NORMAL = {
+    "LocationID": "123456",
+    "LocationName": "test",
+    "SecurityDeviceID": "987654",
+    "PhotoURL": "http://www.example.com/some/path/to/file.jpg",
+    "LocationModuleFlags": "Security=1,Video=0,Automation=0,GPS=0,VideoPIR=0",
+    "DeviceList": None,
+}
+
+LOCATIONS = {"LocationInfoBasic": [LOCATION_INFO_BASIC_NORMAL]}
+
+MODULE_FLAGS = "Some=0,Fake=1,Flags=2"
+
+USER = {
+    "UserID": "1234567",
+    "Username": "username",
+    "UserFeatureList": "Master=0,User Administration=0,Configuration Administration=0",
+}
+
+RESPONSE_AUTHENTICATE = {
+    "ResultCode": 0,
+    "SessionID": 1,
+    "Locations": LOCATIONS,
+    "ModuleFlags": MODULE_FLAGS,
+    "UserInfo": USER,
+}
+
+PARTITION_DISARMED = {
+    "PartitionID": "1",
+    "ArmingState": TotalConnectClient.TotalConnectLocation.DISARMED,
+}
+
+PARTITION_ARMED_STAY = {
+    "PartitionID": "1",
+    "ArmingState": TotalConnectClient.TotalConnectLocation.ARMED_STAY,
+}
+
+PARTITION_ARMED_AWAY = {
+    "PartitionID": "1",
+    "ArmingState": TotalConnectClient.TotalConnectLocation.ARMED_AWAY,
+}
+
+PARTITION_INFO_DISARMED = {}
+PARTITION_INFO_DISARMED[0] = PARTITION_DISARMED
```

Comment:
Please define the constant in one statement when possible.
```py
PARTITION_INFO_DISARMED = {
     0: PARTITION_DISARMED,
}
```

Reply:
done